spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/DirectHandler.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.binder;
+
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageChannel;
+import org.springframework.messaging.MessageHandler;
+import org.springframework.messaging.MessagingException;
+
+/**
+ * @author Marius Bogoevici
+ */
+public class DirectHandler implements MessageHandler {
+
+	private final MessageChannel outputChannel;
```

Comment:
I don't see how it can be. It's the output channel of the handler. Should probably add some JavaDoc (this was moved verbatim from the `MessageChannelBinderSupport`.